clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java

```diff
@@ -284,6 +285,15 @@ private void maybeSendShareAcknowledgeCommitCallbackEvent(Map<TopicIdPartition,
         }
     }
 
+    /**
+     *
+     * @param acknowledgeRequestState Contains the acknowledgements to be sent.
+     * @param currentTimeMs The current time in ms.
+     * @param onCommitAsync Boolean to denote if the acknowledgements came from a commitAsync or not.
+     * @param isAsyncDone Boolean to indicate if the async request has been sent.
```

Comment:
Should this one be called `isAsyncSent` too?

Reply:
Yeah makes sense, I have updated the PR. Thanks.